sdk/data/cosmos/azcosmos_emulator_tests/go.mod

```diff
@@ -0,0 +1,7 @@
+module github.com/Azure/azure-sdk-for-go/sdk/data/cosmos/azcosmos_emulator_tests
+
+go 1.13
```

Comment:
Even though this is not going to be publicly released, this should also be go 1.16

Reply:
Done